partcad-cli/src/partcad_cli/click/command.py

```
@@ -3,6 +3,15 @@
 import partcad as pc
 import coloredlogs
 import logging
+import sentry_sdk
+import rich_click as click
```

Ruff (0.8.2)

Redefinition of unused `click` from line 1

Remove definition: `click`

(F811)